src/components/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,129 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './InlineNotification.module.css';
+import { Icon, Text } from '../../';
+
+export const VARIANTS = ['brand', 'success', 'warning', 'yield'] as const;
+
+const variantToIconAssetsMap: {
+  [key in 'brand' | 'success' | 'warning' | 'yield']: {
+    name: 'info' | 'check-circle' | 'warning' | 'error';
+    title: 'info' | 'success' | 'alert' | 'review';
+  };
+} = {
+  brand: {
+    name: 'info',
+    title: 'info',
+  },
+  success: {
+    name: 'check-circle',
+    title: 'success',
+  },
+  warning: {
+    name: 'warning',
+    title: 'alert',
+  },
+  yield: {
+    name: 'error',
+    title: 'review',
+  },
+};
+
+type Variant = typeof VARIANTS[number];
+
+type Props = {
+  /**
+   * CSS class names that can be appended to the component for styling.
+   */
+  className?: string;
+  /**
+   * Indicates inactive state for the full width variant. Overrides variant prop.
+   * Only to be used with isFullWidth.
+   */
+  inactive?: boolean;
```

Comment:
what's the use case for this?

Reply:
This is for the `inactive` state of the `fullWidth` variant, most right mock in screenshot: 
![image](https://user-images.githubusercontent.com/86632227/169358234-0d47bca9-6f08-43a2-9430-3f4fd26075b0.png)

Reply:
So it's kinda like a disabled state where it's gone extra subtle (no icon and lighter text)? Maybe you could be more explicit in the description for the prop and explain that visual change because this is a pretty unique situation.

Reply:
Sounds good to me, will elucidate in the prop description